phpBB/adm/style/acp_storage.html

```diff
@@ -60,34 +61,23 @@ <h3>{{ lang('WARNING') }}</h3>
 				<fieldset id="{{ storage.get_name }}_{{ provider.get_name }}_settings">
 					<legend>{{ lang('STORAGE_' ~ storage.get_name | upper ~ '_TITLE') }} - {{ lang('STORAGE_ADAPTER_' ~ provider.get_name | upper ~ '_NAME') }}</legend>
 					{% for name, options in provider.get_options %}
-						{% set title = 'STORAGE_ADAPTER_' ~ provider.get_name | upper ~ '_OPTION_' ~ name | upper %}
-						{% set description = 'STORAGE_ADAPTER_' ~ provider.get_name | upper ~ '_OPTION_' ~ name | upper ~ '_EXPLAIN' %}
-						{% set input_id = storage.get_name ~ '_' ~ provider.get_name ~ '_' ~ name %}
-						{% set input_type = options['type'] %}
-						{% set input_name = storage.get_name ~ '[' ~ name ~ ']' %}
-						{% set input_value = attribute(config, 'storage\\' ~ storage.get_name ~ '\\config\\' ~ name) %}
 						<dl>
 							<dt>
-								<label for="{{ input_id }}}">{{ lang(title) }}{{ lang('COLON') }}</label>
+								{% set title = 'STORAGE_ADAPTER_' ~ provider.get_name | upper ~ '_OPTION_' ~ name | upper %}
+								{% set description = 'STORAGE_ADAPTER_' ~ provider.get_name | upper ~ '_OPTION_' ~ name | upper ~ '_EXPLAIN' %}
+								<label>{{ lang(title) }}{{ lang('COLON') }}</label>
 								{% if lang_defined(description) %}
 									<br /><span>{{ lang(description) }}</span>
 								{% endif %}
 							</dt>
 							<dd>
-								{% if input_type in ['text', 'password', 'email'] %}
-									<input id="{{ input_id }}" type="{{ input_type }}" name="{{ input_name }}" value="{{ input_value }}" maxlength="{{ options['maxlength'] ?: 255 }}" />
-								{% elseif input_type == 'textarea' %}
-									<textarea id="{{ input_id }}" name="{{ input_name }}">{{ input_value }}</textarea>
-								{% elseif input_type == 'radio' %}
-									{% for option_name, option_value in options['options'] %}
-										<input type="radio" name="{{ input_name }}" value="{{ option_value }}" class="radio"{% if loop.first %} id="{{ input_id }}"{% endif %}{{ (option_value == input_value) ? ' checked="checked"' }}> {{ lang(option_name) }}
-									{% endfor %}
-								{% elseif input_type == 'select' %}
-									<select name="{{ input_name }}" id="{{ input_id }}">
-										{% for option_name, option_value in options['options'] %}
-											<option value="{{ option_value }}"{{ (option_value == input_value) ? ' selected' }}>{{ lang(option_name) }}</option>
-										{% endfor %}
-									</select>
+								{% set input_name = storage.get_name ~ '[' ~ name ~ ']' %}
```

Comment:
I will improve this block and maybe rewrite the `FormsRadioButtons` macro a little. Don't review this part